nekton/nii2dcm.py

```diff
@@ -0,0 +1,67 @@
+import os
+from pathlib import Path
+
+from base import BaseConverter
+from utils.json_helpers import verify_label_dcmqii_json
+
+
+class Nii2Dcm(BaseConverter):
+    def __init__(
+        self,
+        segmentation_map: Path,
+        out_dcm_format: str = "dicomseg",
+        segmentation_type: str = "multiclass",
+    ):
+
+        supported_output_formats = ["dicomseg"]
+        if out_dcm_format in supported_output_formats:
+            self.format = out_dcm_format
+        else:
+            raise NotImplementedError(
+                f"""for varaible `out_dcm_format` {out_dcm_format} format not implemented.
+                 Choose one from {supported_output_formats}"""
+            )
+        supported_seg_types = ["multiclass", "multilabel"]
```

Comment:
you can define an Enum before the class and use it also the argument in the input, similarly for the output dicom format.

Reply:
Enum is a good idea! This format though will not be used further.

Reply:
completely getting rid of this. The idea is to have a`Nii2DcmSeg` class with 2 different methods for `multilabel`(de-prioritised for a later date) and `multiclass`.